timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/LockWatchValueIntegrationTest.java

```diff
@@ -570,9 +635,9 @@ private void overwriteValueViaKvs(Transaction transaction, Map<Cell, byte[]> val
                 .putUnlessExists(transaction.getTimestamp() - 2, transaction.getTimestamp() - 1);
     }
 
-    private void loadValue() {
+    private void readValueAndAssertLoadedFromRemote() {
         txnManager.runTaskThrowOnConflict(txn -> {
-            txn.get(TABLE_REF, ImmutableSet.of(CELL_1));
+            assertThat(txn.get(TABLE_REF, ImmutableSet.of(CELL_1))).containsEntry(CELL_1, DATA_1);
```

Comment:
sure - I'm not sure what the added check achieves, but no harm

Reply:
Just allows me to extract a more common pattern